panda/src/express/memoryUsage.cxx

```diff
@@ -18,7 +18,7 @@
 #include "mutexImpl.h"
 #include "interrogate_request.h"
 
-#if (defined(WIN32_VC) || defined (WIN64_VC)) && defined(_DEBUG)
+#if defined(_WIN32) && defined(_DEBUG)
```

Comment:
@rdb See my comment on `config_interrogatedb.cxx`

Reply:
This looks like a useful feature.  It should be enabled when Microsoft's debug CRT is used.